main.go

```diff
@@ -54,23 +55,31 @@ type ceTest struct {
 var (
 	namespace          = os.Getenv("POD_NAMESPACE")
 	prometheusEndpoint = os.Getenv("PROMETHEUS_ENDPOINT")
+	env                envConfig
 )
 
 func main() {
-	// listen on port 8080 for any event
+	/**
+	Note that we open two ports, 8080 and env.Port (e.g., 8082).
+	Everything goes in via 8080, but if it's a cloudevent, we forward it internally to 8082
```

Comment:
What is the reason behind to have thee different ports right now ? 

Also what is the Job of Port `8080`
* Proxy Cloud Events 
* Being Reachable for the Alertmanager WebHook Call

The `8080` seems to be the ingress (proxy for cloud right now and `8082` is the cloud events and prometheus webhook call)

Reply:
I think it would be a nice improvement to mention the responsibilities of the Ports also in the README.
Then it is more clear to external users what the port is capable of.

Reply:
Port 8081 is taken by the Keptn distributor within the sidecar.
Port 8080 is the primary port that this pod is listening too (and the only one a user needs to know about)
Port 8082 is for internal usage, it takes cloudevents using the cloudevents sdk.

I'd like to get rid of 8082, but that's a different story and requires refactoring way beyond the scope of this bugfix.

I'll update the information in README though.

Reply:
Updated PR